src/Cli/Microsoft.DotNet.Cli.Utils/Verbosity.cs

```diff
@@ -0,0 +1,31 @@
+namespace Microsoft.DotNet.Cli.Utils;
+
+/// <summary>
+/// Represents the desired verbosity level for command output.
+/// Maps mostly to MSBuild's verbosity levels.
+/// </summary>
+public enum Verbosity
+{
+    quiet,
+    minimal,
+    normal,
+    detailed,
+    diagnostic
```

Comment:
I know I asked this in the past, but I forgot the answer. Is it necessary for these to be lowercase? It is just anti-C# style to have public things be entirely lowercase.